scripts/resultsSum.pl

```diff
@@ -156,7 +156,25 @@ sub resultReporter {
 						$tapString .= "not ok " . $numOfTotal . " - " . $result . "\n";
 						$tapString .= "  ---\n";
 						if (($diagnostic eq 'failure') || ($diagnostic eq 'all')) {
-							$tapString .= $output;
+							if ($buildList =~ /openjdk/) {
+							    my @lines = split('\\n', $output);
+								my $failureTests = "";
+								foreach my $line (@lines)
+								{
+									if ( $line =~ /TEST: /) {
```

Comment:
Is it possible to match the 50 hyphens before and then the `TEST:`
```
11:44:10  --------------------------------------------------
11:44:12  TEST: java/util/stream/test/org/openjdk/tests/java/util/stream/FilterOpTest.java
```
This could be more reliable compared to only `TEST:`. In an extreme case, if we have not only openjdk but also other folders in the buildlist, it could be more problematic.

Reply:
Updated. Trying with multiple line query doesn't work well ( works in my local environment and failed with builds). Decided just using the simple loop, which works and feels simple and straightforward.